src/calculator.css

```diff
@@ -42,11 +53,276 @@ li.nav-links:first-child > .active-link {
 }
 
 li.nav-links:last-child {
-  margin: 0 20px 0 auto;
+  margin: 0 30px 0 auto;
   border: 1px solid white;
   border-radius: 100px;
 }
 
 .login {
   margin: 2px;
 }
+
+/* end nav section */
+
+/* start input book section */
+
+.wrapper-input {
+  display: flex;
+  flex-wrap: nowrap;
+  height: 2.813rem;
+}
+
+.book-title-input {
+  flex: 2 1 auto;
+  padding: 10px;
+  border-radius: 4px;
+  border: 1px solid gray;
+}
+
+.book-category-input {
+  flex: 1 1 auto;
+  margin: 0 2.125rem;
+  padding: 10px;
+  border-radius: 4px;
+  border: 1px solid gray;
+}
+
+.book-submit {
+  flex: 0.5 1 auto;
+  background-color: rgb(75, 75, 233);
+  border: 1px solid rgb(101, 99, 211);
+  border-radius: 4px;
+  color: inherit;
+  font-weight: bolder;
+  cursor: pointer;
+}
+
+/* end input book section */
+
+/* start book section */
+
+.book-list-item {
+  display: flex;
+  flex-wrap: nowrap;
+  padding: 20px;
+  background-color: rgb(0, 0, 0);
+  margin: 0 0 20px 0;
+}
+
+.book-info {
+  flex: 2 1 auto;
+  display: flex;
+  flex-direction: column;
+}
+
+.book-progress {
+  flex: 1 1 auto;
+  display: flex;
+  align-items: center;
+}
+
+.book-chapteer {
+  flex: 1 1 auto;
+  display: flex;
+  flex-direction: column;
+}
+
+.book-title {
+  font-size: 22px;
+  padding: 10px 0;
+}
+
+.book-category {
+  font-size: 16px;
+}
+
+.crud-operations {
+  margin-top: 20px;
+}
+
+.crud {
+  margin: 0 20px 0 0;
+  padding: 0;
+  background-color: rgb(0, 0, 0);
+  border: none;
+  color: rgb(75, 75, 233);
+  cursor: pointer;
+}
+
+/* end book section */
+
+/* start progress section */
+.padding {
+  padding: 10rem !important;
```

Comment:
- [ ] Please, don't use !important tool on CSS style, is a bad practice, and this breaks the CSS normal flow. 🐒

Reply:
Thank you for taking the time to leave comments.